manifests/0000_30_cluster-baremetal-operator_05_deployment.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cluster-baremetal-operator
+  namespace: openshift-machine-api
+  labels:
+    k8s-app: cluster-baremetal-operator
+  annotations:
+    exclude.release.openshift.io/internal-openshift-hosted: "true"
```

Comment:
I am not sure what the significance of this annotation is. Some of our manifests have it and others don't. Is there some documentation about this annotation?

Reply:
Looks like it's the _cluster profile_ is used to exclude operators from being applied when the control plane is externally hosted (such as in the case of IBM Cloud, as described in [1]; see also [2]).
So I think we should keep it.

[1] https://github.com/openshift/enhancements/pull/200
[2] https://github.com/openshift/cluster-version-operator/pull/269